core/trino-main/src/main/java/io/trino/server/ServerConfig.java

```diff
@@ -104,4 +121,10 @@ public ServerConfig setQueryInfoUrlTemplate(String queryInfoUrlTemplate)
         this.queryInfoUrlTemplate = Optional.ofNullable(queryInfoUrlTemplate);
         return this;
     }
+
+    public enum CoordinatorShutdownStrategy
+    {
+        PREEMPT,
```

Comment:
These values are very vague, how about `CANCEL_QUERIES` and `WAIT_FOR_RUNNING_QUERIES`?